src/main/java/com/pagsestagio/movieapi/controller/resposta/FilmeRespostaRetornaFilmeOuMensagem.java

```diff
@@ -1,15 +1,19 @@
 package com.pagsestagio.movieapi.controller.resposta;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.UUID;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public record FilmeRespostaRetornaFilmeOuMensagem(
-    UUID idpublico,
-    String nomeFilme,
-    String sinopseFilme,
-    String categoriaFilme,
-    Integer anoFilme,
-    String diretorFilme,
-    String mensagemStatus)
+        @JsonProperty("idPublico") UUID idpublico,
```

Comment:
porque você colocou essas anotações?

Reply:
Foi a forma que arrumei para evitar de aparecer o "mensagemStatus: null" quando a requisição for feita com sucesso, aparece só as coisas relevantes, da mesma forma, não vai aparecer as infos sobre o filme quando der problema, só a mensagem de erro. Só fiquei refletindo que se eu puxo as infos da api externa, e ela em algum dos campos retorna null pro campo sinopse por exemplo, sinopse nem ia aparecer, não seria um campo visivel ali, mas lembro de fazer um teste que acho que aparece um N/A, algo do tipo, então null acho que não retorna